# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysstat/linux26.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- sysstat-5.1.3/ioconf.c.vanilla	2004-12-26 02:42:36.470729288 +0100
+++ sysstat-5.1.3/ioconf.c	2004-12-26 02:43:10.872499424 +0100
@@ -27,6 +27,10 @@
 #include <errno.h>
 #include <linux/major.h>
 
+#ifndef MAX_BLKDEV
+#define MAX_BLKDEV      255
+#endif
+
 #include "ioconf.h"
 
 #ifdef USE_NLS
